README.md

```diff
@@ -329,13 +329,32 @@ If you have classes with a lot of columns and you filter them often with the sam
           {
             "name": "email",
             "filterSortToTop": true
-          }          
+          }
         ]
       }
     }
 ]
 ```
 
+### Persistent Filters
+
+The `classPreference` setting allows for data browser filters to be saved across sessions. To save the current filters, navigate to *App Settings > Export Class Preferences*.
+
+For example:
+
+```json
+"apps": [{
+  "classPreference": {
+    "_Role": {
+      "filters": [{
+        "name": "Filter Name",
+        "filter": "[{\"field\":\"objectId\",\"constraint\":\"exists\"}]"
```

Comment:
About the filter value, why does its value need to be escaped? Why couldn't we just pass an object that would make it also more readable.